tests/GPSR_appendix.tex

```diff
@@ -0,0 +1,88 @@
+
+\section{Command retrieval explained}
+The robot has to show it has understood te given command by stating all the required information to accomplish the task. For this purpose, the robot may repeat the understood command and ask for confirmation. It is not required to repeat the command word by word; reprasing the command is allowed. For instance, if the robot is instructed to \quotes{place a coke onto the tray}, the robot may either say: \textit{\quotes{You want me to place a coke on the tray. Is that correct?}} or \textit{\quotes{do you want me to deliver a coke to the tray?}}.
+
+If The robot can't correctly recognize the given command, it is allowed to request the operator to repeat the command up to three times. After three failed attempts, a new command is generated. Th team may opt to use a custom operator or the Continue rule (Section Section 3.8.15).
+
+When a robot has partially understood the command, it is allowed to ask the operator for additional information (e.g. \textit{\quotes{did you say apple juice or pineapple juice?}}).
+
+\subsection{Missing information}
+When a given command lacks of information required for accomplishing the task, the robot should request for that missing part. For instance, if the robot is instructed to \textit{\quotes{offer a drink to the person at the door}}, it may ask \textit{\quotes{which drink should I deliver to the person at the door?}} It is also possible that the robot simply confirms the command and takes a random drink from the drinks location, but in those cases, the command will be considered by the jury of an inferior category.
+
+\subsection{Wrong information}
+Some Category III commands contains erroneous information. In these cases, the robot should
+\begin{itemize}
+	\item be able to realize such an error while trying to carry out the task, get back to the operator, and clearly state why it wasn't able to accomplish the task; or
+	\item be able to solve the problem by means of an alternative, reasonable solution.
+\end{itemize}
+
+For example, lets assume the robot is commanded to \textit{\quotes{move the organge juice from the fridge to the dinner table}}, but in the fridge there are only the apple juice and the milk, while the orange juice lies in the stove. The robot may either explain to the operator that there are no orange juices in the fridge, or search the kitchen for the orange juice, grasp it from the stove and deliver it to the dinner table.
+
+\section{Command categories explained}
+All possible actions has been classified previously by the TC according to their difficulty. For each command the team may choose from the following three categories:
+
+\subsection{Category I}
+This category comprehends easy-to-solve tasks with a low difficulty degree, involving indoor navigation, grasping known objects, answering questions (from the predefined set of questions), etc.
```

Comment:
Neither of them. Questions about the crowd make no sense since there is no crowd, but the questions about the arena, objects, etc, are stll valid and this is GPSR. The correct answer is **A - {crowd}**